iib/workers/tasks/legacy.py

```diff
@@ -120,7 +123,9 @@ def _push_package_manifest(package_dir, cnr_token, organization):
         app registry via OMPS.
     :param str organization: the organization name in the legacy app registry to which
          the backported packages should be pushed to.
-    :raises IIBError: if the push is unsucessful
+    :return: Response from OMPS in JSON format
+    :rtype: dict|None
+    :raises IIBError: if the push is unsuccessful
```

Comment:
Optional: `if the push is unsuccessful` => `if the push fails`

Reply:
I guess your suggestion is better